astropy/tests/helper.py

```diff
@@ -304,14 +304,27 @@ def _save_coverage(cov, result, rootdir, testing_path):
     # Python 2.x.
```

Comment:
Agreed most of this could probably be removed in the near future, but we haven't formally deprecated 2to3 usage I don't think.